core/scripts/chaincli/command/chain_table_cleanup.go

```diff
@@ -0,0 +1,105 @@
+package command
+
+import (
+	"fmt"
+	"github.com/smartcontractkit/chainlink/v2/core/services/chainlink"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+	"github.com/smartcontractkit/chainlink/v2/core/store/dialects"
+	"github.com/smartcontractkit/sqlx"
+	"github.com/spf13/cobra"
+	"strings"
+)
+
+var (
+	chainType     string
+	chainID       int
+	clDatabaseURL string
+)
+
+var tablesToCleanup = map[string][]string{
+	"EVM": {"evm_log_poller_filters",
+		"evm_log_poller_blocks",
+		"log_broadcasts",
+		"block_header_feeder_specs",
+		"direct_request_specs",
+		"evm_logs",
+		"vrf_specs",
+		"evm_heads",
+		"evm_forwarders",
+		"blockhash_store_specs",
+		"evm_key_states",
+		"log_broadcasts_pending",
+		"eth_txes",
+		"keeper_specs",
+		"flux_monitor_specs",
+		"ocr_oracle_specs"},
+}
```

Comment:
How are we ensuring this is kept up-to-date?

This used to be handled cleanly and automatically by foreign keys, which is why we had the table in the first place.

Reply:
Yeah, this is the main concer. It would be needed to be updated manually. 
Alternatively I can crawl the database and find all tables that have evm_chain_Id and clear those, not sure if that could cause some problems